dask_planner/src/dialect.rs

```diff
@@ -1,6 +1,6 @@
 use core::iter::Peekable;
 use core::str::Chars;
-use sqlparser::dialect::Dialect;
+use datafusion_sql::sqlparser::dialect::Dialect;
```

Comment:
We can use the version of sqlparser re-exported by DataFusion rather than manage this dependency ourselves

Reply:
Great!